learn/getting_started/filtering_and_sorting.md

```diff
@@ -18,7 +18,7 @@ The `settings` object contains two arrays for this purpose: `filterableAttribute
 
 The above code sample adds `mass` and `_geo` to `filterableAttributes` and `sortableAttributes` setting you up for the next sections.
 
-To learn more about the `settings` object and how to configure it, refer to our [dedicated guide](/learn/configuration/settings.md).
+To learn more about the `settings` object and how to configure it, refer to our [dedicated guide](/reference/api/settings.md).
```

Comment:
"Dedicated guide" is not just go-to link text; it actually means something. We want our users to understand what they're clicking on.

```suggestion
To learn more about the `settings` object and how to configure it, check out the [API reference](/reference/api/settings.md).
```